products/messaging/frontend/FunctionsTable.tsx

```diff
@@ -65,7 +65,11 @@ export function FunctionsTable({ type, kind }: FunctionsTableProps): JSX.Element
                             render: function RenderPluginName(_, hogFunction) {
                                 return (
                                     <LemonTableLink
-                                        to={urls.hogFunction(hogFunction.id)}
+                                        to={
```

Comment:
Same here - lets not muddy this. We know it wont be a hog function so this is just going to make other tidying efforts harder